### DISCUSSION TITLE
Fix the queuing time valve!

### DISCUSSION THREAD

**Discussion by Psy7en** [ValveSoftware/Dota2-Gameplay#9439] [General]
It takes more than 20 minutes to find a game in Dota now. There is some issue with the matchmaking algorithm, as Im only getting highly varied player behavior score matches. Please fix this valve.

**Comment by richardfox00123** [comment]
Matchmaking calculations are perplexing frameworks that game designers persistently refine to adjust factors like player ability, conduct, and accessibility. Assuming you're experiencing expanded matchmaking times or reliably getting profoundly changed player conduct scores, it's conceivable that there might be an issue or change required in the matchmaking framework.

To address this worry, I prescribe contacting the Dota 2 local area discussions, official help channels, or other local area stages where players and engineers lock in. By sharing your input and encounters, you can add to the continuous improvement of the game and assist with bringing issues to light of the issue. [YourTexasBenefits](https://www.yourtexasbenefits.bid/)

Valve depends on player criticism to recognize and address matchmaking issues, so it's fundamental to impart your interests straightforwardly to them. They can examine what is going on, assemble information, and make changes in accordance with upgrade the matchmaking experience for all players.

I hope that your concerns are addressed soon, and you can enjoy improved matchmaking in Dota 2.
Thank if you are like my answer.
